haskell/nettest/Nettest.hs

```diff
@@ -54,6 +95,160 @@ scTransferScenario constructInitialStorage contract_ = uncapsNettest $ do
     callTransfer from to value =
       callFrom (AddressResolved from) sc (ep "transfer") (tp from to value)
 
-  comment "Transfer from superuser to superuser"
-  callTransfer superuser superuser 111
-  callTransfer superuser owner1 20
+    callTransfers
+      :: [("from_" :! Address, [("to_" :! Address, "amount" :! Natural)])]
+      -> capsM ()
+    callTransfers = mapM_ $ \(from@(arg #from_ -> from_), destinations) ->
+      callFrom
+        (AddressResolved from_)
+        sc
+        (ep "transfer")
+        (constructTransfersFromSender from destinations)
+
+    configureMinter :: Address -> Address -> Maybe Natural -> Natural -> capsM ()
+    configureMinter from for expectedAllowance newAllowance =
+      callFrom (AddressResolved from) sc (ep "configure_minter") (#minter .! for, (#current_minting_allowance .! expectedAllowance, #new_minting_allowance .! newAllowance))
+
+    removeMinter :: Address -> Address -> capsM ()
+    removeMinter from whom = callFrom (AddressResolved from) sc (ep "remove_minter") whom
+
+    mint :: Address -> Natural -> capsM ()
+    mint to_ value = callFrom (AddressResolved to_) sc (ep "mint") [(#to_ .! to_, #amount .! value)]
+
+    burn :: Address -> Natural -> capsM ()
+    burn from amount_ = callFrom (AddressResolved from) sc (ep "burn") [amount_]
+
+    pause :: Address -> capsM ()
+    pause from = callFrom (AddressResolved from) sc (ep "pause") ()
+
+    unpause :: Address -> capsM ()
+    unpause from = callFrom (AddressResolved from) sc (ep "unpause") ()
+
+    transferOwnership :: Address -> Address -> capsM ()
+    transferOwnership from to = callFrom (AddressResolved from) sc (ep "transfer_ownership") to
+
+    acceptOwnership :: Address -> capsM ()
+    acceptOwnership from = callFrom (AddressResolved from) sc (ep "accept_ownership") ()
+
+    changeMasterMinter :: Address -> Address -> capsM ()
+    changeMasterMinter from to = callFrom (AddressResolved from) sc (ep "change_master_minter") to
+
+    changePauser :: Address -> Address -> capsM ()
+    changePauser from to = callFrom (AddressResolved from) sc (ep "change_pauser") to
+
+    setSafelist :: Address -> Address -> capsM ()
+    setSafelist from safelistAddress = callFrom (AddressResolved from) sc (ep "set_safelist") (Just safelistAddress)
+
+    unsetSafelist :: Address -> capsM ()
+    unsetSafelist from = callFrom (AddressResolved from) sc (ep "set_safelist") (Nothing :: Maybe Address)
+
+  let
+    transferScenario = do
+      comment "Transfer from superuser to superuser "
```

Comment:
IMO it looks a bit confusing when you print `"Transfer from superuser to superuser"`, make a transfer from superuser to superuser and then make other transfers before printing something else. If `callTransfer superuser owner1 40` fails, one will think that "Transfer from superuser to superuser failed", but in reality a different transfer failed. So IMO if there is a comment `X` then all actions between `X` and the next comment `Y` (if there is `Y`) should somehow relate to `X`.

Reply:
Can you change it to "Transfers from superuser"? Because there are 2 transfers in this group, both are from `superuser`, but only one is to `superuser`.